package.json

```diff
@@ -36,7 +36,7 @@
     "react-shadow": "^17.6.0",
     "react-spatial": "0.2.14",
     "react-styleguidist": "^11.0.4",
-    "react-transit": "^0.3.1",
+    "react-transit": "^0.3.2-beta.1",
```

Comment:
Before merging, use a patch version of react-transit as soon as https://github.com/geops/react-transit/pull/280 is merged.